board/sunxi/cubie_board.c

```diff
@@ -7,20 +7,34 @@
 /*PK 24 */
 #define HW_ID_GPIO			344
 
+enum board_type {
+	BOARD_A7A,
+	BOARD_A7Z,
+	BOARD_A7S
+};
+
 struct hw_info_def {
 	char *compatible;
 	char *fdtfile;
+	const char* *pcie_pins;
 	unsigned int hw_id_level;
 	unsigned int hw_id_lower_bound;
 	unsigned int hw_id_upper_bound;
 };
 
+enum board_type board_type = BOARD_A7A;
+
+static const char *pcie_pin_a7a[] = {"PE11", "PE12", "PE13" };
+static const char *pcie_pin_a7z[] = {"PD20", "PD21", "PD22" };
+static const char *pcie_pin_a7s[] = {"PD20", "PD21", "PD22" };
+
 static struct hw_info_def hw_info[] = {
```

Comment:
```suggestion
static const struct hw_info_def hw_info[] = {
```
现在会把指针传出去，这边预防性设置成常量